cwatch.php

```diff
@@ -348,7 +348,23 @@ public function addService(
             );
         }
 
-        $return = [['key' => 'cwatch_licenses', 'value' => $license_keys, 'encrypted' => 0]];
+        // Get user customer ID
+        $api = $this->getApi();
+
+        $customer_id = null;
+        $user_response = $api->getUser($vars['cwatch_email']);
+        if ($user_response->status() == 200 && ($users = $user_response->response())) {
+            $customer_id = $users[0]->id;
```

Comment:
Since this is user input from the API you might check that `$users[0]->id` exists first